### PR TITLE
Docstrings for orthogonal polynomial transforms

### PR DESCRIPTION
**Pull request by jishnub**
I've tried to add docstrings for some of the transforms. More can be added later.

### CODE REVIEW

**Comment by dlfivefifty** [comment]
Please remove the `::StridedVector`. If those restrictions are in the code they should be removed.

**Comment by jishnub** [comment]
I'm not sure if I follow: are you suggesting changing it to `AbstractVector` or to `Vector`? In the former case, I'm not certain if the library can support arbitrary Julia vectors (e.g. ranges). In the latter, I think it does work with strided vectors (e.g. views)

File: src/docstrings.jl [hunk 1]
```diff
@@ -0,0 +1,121 @@
+"""
+	leg2cheb(v::StridedVector; normleg::Bool=false, normcheb::Bool=false)
```
Comment (dlfivefifty) [inline src/docstrings.jl:2]:
```suggestion
	leg2cheb(v::AbstractVector; normleg::Bool=false, normcheb::Bool=false)
```
Also below

**Comment by jishnub** [comment]
Bump, is any any other change necessary here?

**Comment by MikaelSlevinsky** [comment]
Fine with me

**Comment by jishnub** [comment]
Good to merge?

**Comment by jishnub** [comment]
Gentle bump

**Pull request merged**